apps/tmux/tmux.talon

```diff
@@ -0,0 +1,9 @@
+tag: user.tmux
+-
+tag(): user.splits
+
+# pane management - these commands use the word split to match with the splits
+# tag defined in tags/splits/splits.talon
+split move <user.arrow_key>: user.do_tmux_keybind(arrow_key)
++#Say a number after this command to switch to pane
```

Comment:
Seems to be an extra + on these two lines?

Reply:
thanks for pointing that out 😅 fixed